.docs/code-design.md

```
@@ -1105,6 +1105,18 @@ cleanupExpiredMigrationsInternal:
 | Yahoo!テキスト解析API         | サービス名からのふりがな自動生成             | アプリケーションID                             |
 | Cloudflare Workers / R2 | Convexデータの定期自動バックアップ（日次）   | Cloudflare Secret（`CONVEX_DEPLOY_KEY`） |
 
+## 4.3 オンボーディング関連変更
+
+### users テーブル拡張
+| フィールド | 型 | 説明 |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)